psignifit/psignifit.py

```diff
@@ -13,7 +13,7 @@
 from ._priors import setup_priors
 from ._result import Result
 from ._typing import ParameterBounds, Prior
-from ._utils import (PsignifitException, check_data)
+from ._utils import PsignifitException, check_data, cast_np_scalar
```

Comment:
alphabetical order?

Reply:
fixed in 326c7caa01205a68d86463c5bc55402a90b1999a